File: CS/T258870/Service/IReportViewerService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ReportServer.ServiceModel.ConnectionProviders;
using DevExpress.Xpf.Printing;

namespace T258870.Service {
    public interface IReportViewerService {
        ConnectionProvider connectionProvider { get; set; }
        void Show(int reportId);
    }
}

File: VB/T258870/Service/IReportViewerService.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.ReportServer.ServiceModel.ConnectionProviders
Imports DevExpress.Xpf.Printing

Namespace T258870.Service
    Public Interface IReportViewerService
        Property connectionProvider() As ConnectionProvider
        Sub Show(ByVal reportId As Integer)
    End Interface
End Namespace
